app/assets/javascripts/global_activities/side_pane.js

```diff
@@ -191,20 +198,22 @@ $(function() {
   $('#calendar-to-date').on('dp.change', function(e) {
     var dateContainer = $('.ga-side .date-selector.filter-block');
     if (!updateRunning) {
-      $('#calendar-from-date').data('DateTimePicker').minDate(e.date);
+      $('#calendar-from-date').data('DateTimePicker').maxDate(e.date);
```

Comment:
Activities are fetched in reverse chronological order

Reply:
Ok, I'm tad confused here about the names. Not really sure if this is something intuitive for the user. And in epics it also is FROM-TO. Should then also the filter label be reversed as it was (`Activity created: 31.01.2019-01.01.2019`)?
I'm willing to bet someone's gonna burn himself on the names down the road. :D

Reply:
User is not supposed to look into HTML attributes. User will still see 'start' and 'end' date.

Reply:
I meant for developer fixing bugs. For user it's intuitive to think 2 date fields will be from-to, e.g. you'll probably never say 'until today from yesterday', but what do I know, I'm just a developer.

Reply:
where did you take 'until today from yesterday' from? What is illogical in 'from today to previous Sunday' as we use reverse chronological order?